mission_control/Mission mcMission
---
mavros_msgs/WaypointList mrMission